pkg/plugins/models.go

```diff
@@ -288,3 +290,9 @@ type Permission struct {
 	Action string `json:"action"`
 	Scope  string `json:"scope"`
 }
+
+type EnterpriseCachingConfig struct {
```

Comment:
Nit. Maybe `QueryCachingConfig` is a more suitable name?

Reply:
[a9877c6](https://github.com/grafana/grafana/pull/61968/commits/a9877c60483ea51669d429c9e882933da0903ae7)